firefox-ios/Client/Frontend/TabContentsScripts/FormAutofillHelper/FormAutofillHelper.swift

```diff
@@ -165,8 +165,41 @@ class FormAutofillHelper: TabContentScript {
 
         return AutofillFieldValuePayload(fieldValue: .address, fieldData: addressPlainText)
     }
+
     // MARK: - Injection
 
+    static func injectAddressInfo(logger: Logger,
```

Comment:
Why static? Can we avoid it?

Reply:
@lmarceau this is one area where avoiding static is not trivial but due to nature of how autofill works, the injection can take place from practically anywhere. Here in this case the flow is `address input form tap -> show accessory view -> bottom sheet -> tap on an address`

Due to the nature of this flow by the time tapping on address happens the only way to trigger an injection is from within that Tab. Which seems feasible however adding more scripts makes it more convoluted inside the tab. 

So even though static might not be the right variable the argument is to put it in one place where it belongs. Here it `best` belongs inside FormAutofillHelper.swift compared to Tab.swift

I am open for suggestion :)

Reply:
@jnrahme On a separate note, did we add tests for this? 
Sorry I might have asked before but are you are waiting for any js fixes for this area?

Reply:
Added tests